packages/expo-asset/src/Asset.ts

```diff
@@ -159,10 +159,9 @@ export class Asset {
         height: meta.height,
       });
 
-      // TODO: FileSystem should probably support 'downloading' from drawable
-      // resources But for now it doesn't (it only supports raw resources) and
-      // React Native's Image works fine with drawable resource names for
-      // images.
+      // For images backward compatibility,
+      // keeps localUri the same as uri for React Native's Image that
+      // works fine with drawable resource names.
```

Comment:
though i did add drawables support in this pr, but i was hesitant that would introduce breaking changes. let's keep it as-is for drawables (images).